networks/harmonytestnet.config.ts

```diff
@@ -52,6 +53,11 @@ export const harmonytestnet: NetworkUserConfig = {
         name: TOKEN_NAMES.USDC,
         address: '0x2e89d104f10e1d480d47e924f3f43672c09d12a2',
       },
+      {
+        factory: EthereumERC20__factory,
+        name: TOKEN_NAMES.WONE,
+        address: '0x7466d7d0c21fa05f32f5a0fa27e12bdc06348ce2',
```

Comment:
Somehow verified at https://explorer.testnet.harmony.one/address/0x7466d7d0c21fa05f32f5a0fa27e12bdc06348ce2

But testnet isn't a blocker for deployment.